org.osgi.test.junit5/pom.xml

```diff
@@ -136,9 +140,21 @@
 								-make: (*).(jar); type=bnd; recipe="${.}/bnd/$1.bnd"
 								-includeresource:\
 									tb1.jar,\
-									foo/tbfoo.jar=tb1.jar
+									beforeClass.jar,\
```

Comment:
same question about all the embedded jars with the same contents. Do we really need them?

Reply:
comments?

Reply:
Done, per above.